packages/shared/warningWithoutStack.js

```diff
@@ -15,8 +15,6 @@
 let warningWithoutStack = () => {};
 
 if (__DEV__) {
-  const consoleError = Function.prototype.bind.call(console.error, console);
-
   warningWithoutStack = function(condition, format, ...args) {
```

Comment:
Let’s assert the number of arguments isn’t higher than supported here?

Reply:
As far as I can see the maximum of 8 arguments is used at the moment for this warning. Do you want to add extra check? But how many arguments do we need to check? That's exactly the reason I didn't use this approach initially..

btw, the tests marked as failed, but as I can see on circleci - tests itself passed, but later this happened. Is it related to the change? Or verification job could be just restarted?

Request failed [401]: https://api.github.com/repos/facebook/react/pulls/13620/requested_reviewers
Response: {
  "message": "Bad credentials",
  "documentation_url": "https://developer.github.com/v3"
}
Error:  TypeError: Cannot read property 'repo' of undefined
    at GitHub.APIMetadataForPR (/home/circleci/project/node_modules/danger/distribution/platforms/GitHub.js:274:27)
    at GitHub.<anonymous> (/home/circleci/project/node_modules/danger/distribution/platforms/GitHub.js:164:39)
    at step (/home/circleci/project/node_modules/danger/distribution/platforms/GitHub.js:40:23)
    at Object.next (/home/circleci/project/node_modules/danger/distribution/platforms/GitHub.js:21:53)
    at fulfilled (/home/circleci/project/node_modules/danger/distribution/platforms/GitHub.js:12:58)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
Danger failed

Reply:
If 8 is current maximum the let’s check we have at most 8.

Reply:
Try rebasing to solve the CI issue